frontend/src/lobby/NameInput.tsx

```diff
@@ -0,0 +1,49 @@
+import { Component, createSignal } from "solid-js";
+
+import { userName, setUserName } from "./PlayerList"
+import { setUserNameMsg } from "../SetUserName";
+import { sendMessage } from "../App";
+
+
+const NameInput: Component = () => {
+  const [tempUsername, setTempUsername] = createSignal<string>(userName())
```

Comment:
we should commit to either always writing UserName or Username otherwise it will lead to bugs one day 😆